sufia-models/app/services/sufia/characterization_service.rb

```diff
@@ -0,0 +1,67 @@
+module Sufia
+  class CharacterizationService
+    include Hydra::Derivatives::ExtractMetadata
+
+    delegate :mime_type, :uri, to: :@generic_file
+    attr_reader :generic_file
+
+    def self.run(generic_file)
```

Comment:
I've seen service objects use different methods as their primary interface, usually `run`, `call`, and `create`. Is it worth converging on a convention here?